packs/equipment-effects/effect-bloodseeker-beak.json

```diff
@@ -21,11 +21,48 @@
             "title": "Pathfinder GM Core"
         },
         "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "predicate": [
+                            {
+                                "not": "parent:origin:item:slug:bloodseeker-beak"
+                            }
+                        ],
+                        "value": "Compendium.pf2e.equipment-srd.Item.Bloodseeker Beak (Greater)"
+                    },
+                    {
+                        "predicate": [
+                            {
+                                "not": "parent:origin:item:slug:bloodseeker-beak-greater"
+                            }
+                        ],
+                        "value": "Compendium.pf2e.equipment-srd.Item.Bloodseeker Beak"
+                    }
+                ],
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.EquipmentVariety",
+                "rollOption": "bloodseeker-beak"
+            },
             {
                 "category": "precision",
                 "diceNumber": 1,
                 "dieSize": "d4",
                 "key": "DamageDice",
+                "predicate": [
+                    "bloodseeker-beak:bloodseeker-beak"
+                ],
+                "selector": "strike-damage"
+            },
+            {
+                "category": "precision",
+                "diceNumber": 2,
+                "dieSize": "d4",
+                "key": "DamageDice",
+                "predicate": [
+                    "bloodseeker-beak:bloodseeker-beak-greater"
+                ],
                 "selector": "strike-damage"
             }
```

Comment:
This could use a `hideIfDisabled` for a cleaner roll dialog. There's no reason more than one should be visible in this case.